clouddriver-cloudrun/src/main/java/com/netflix/spinnaker/clouddriver/cloudrun/deploy/ops/DeployCloudrunAtomicOperation.java

```diff
@@ -77,6 +81,12 @@ public String deploy(String repositoryPath) {
             description.getFreeFormDetails(),
             description.getSuppressVersionString());
     List<String> configFiles = description.getConfigFiles();
+    try {
+      populateCloudrunYmlData(configFiles);
+    } catch (Exception e) {
+      throw new CloudrunOperationException(
+          "Failed to deploy to Cloud Run with command " + e.getMessage());
```

Comment:
Stack trace is swallowed here, can we log it?

Reply:
logged it....done @mattgogerly

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?